src/components/commons/Button.tsx

```diff
@@ -36,26 +36,27 @@ export default function Button({
   const handleButtonClick = () => {
     if (onClick) {
       onClick();
-      return;
     }
-    window.open(href, "_blank", "noreferrer");
   };
 
   const transitionStyle = "transition-all ease-in duration-300";
+
   return (
-    <button
-      data-testid={`${testId}-button`}
-      type="button"
-      className={`flex items-center justify-center rounded-[28px] group border border-white-50 brand-bg-gradient-1 active:brand-bg-gradient-2 hover:border-transparent
+    <a href={href}>
```

Comment:
if we move `<a>` here, does `disable` still work?

Reply:
if we put the `<a>` here the button wouldnt redirect